src/libclipper/include/clipper/persistent_state.hpp

```diff
@@ -25,7 +25,7 @@ size_t state_key_hash(const StateKey& key);
 // Threadsafe, non-copyable state storage
 class StateDB {
  public:
-  StateDB();
+  explicit StateDB();
```

Comment:
Good catch. I added a single-argument constructor then deleted it and forgot to remove explicit.